src/cache/precomputed_quantities.jl

```diff
@@ -366,7 +383,59 @@ function set_precomputed_quantities!(Y, p, t)
                     ᶜtsʲ,
                     ᶜspecificʲ.e_tot,
                 )
+
+                # Overwriting the first interior point for EDMFX ᶜh_totʲ
+                # based on boundary conditions for updrafts
+                ᶜh_tot_int = Fields.level(ᶜh_tot, 1)
+                ᶜh_totʲ_int = Fields.level(ᶜh_totʲ, 1)
+                @. ᶜh_totʲ_int = sgs_h_tot_first_interior_bc(
+                    $Fields.field_values(ᶜz_int),
```

Comment:
Note to myself: double check if this should be `z_int` or `z_int - z_sfc`

Reply:
`z_int - z_sfc` looks more reasonable to me. Should I change it?

Reply:
Sounds good, thanks!